hookman/hookman_generator.py

```diff
@@ -310,7 +310,7 @@ def _hook_specs_header_content(self, shared_lib_name) -> str:
         #ifndef {self.project_name.upper()}_HOOK_SPECS_HEADER_FILE
         #define {self.project_name.upper()}_HOOK_SPECS_HEADER_FILE
         #ifdef WIN32
-            #define HOOKMAN_API_EXP __declspec(dllexport)
+            #define HOOKMAN_API_EXP extern "C" __declspec(dllexport)
             #define HOOKMAN_FUNC_EXP __cdecl
         #else
             #define HOOKMAN_API_EXP
```

Comment:
I think that you need to add the extern "C" on line 316 for Linux as well
```c
 #define HOOKMAN_API_EXP extern "C"
```

Reply:
Oops good catch